app.py

```diff
@@ -11,18 +11,27 @@
 db = SQLAlchemy(app)
 api = Api(app, title="Review Requester")
 
+
+reviewer_languages = db.Table('reviewer_languages',
+    db.Column('reviewer_id', db.Integer, db.ForeignKey('reviewer.id'), primary_key=True),
+    db.Column('review_language_id', db.Integer, db.ForeignKey('review_language.id'), primary_key=True)
+)
+
+
 class Reviewer(db.Model):
     __tablename__ = 'reviewer'
 
     id = db.Column(db.Integer, primary_key=True)
-    first_name = db.Column(db.String(50), unique=True, nullable=False)
-    last_name = db.Column(db.String(50), unique=True, nullable=False)
+    first_name = db.Column(db.String(50), nullable=False)
+    last_name = db.Column(db.String(50), nullable=False)
     email_address = db.Column(db.String(50), unique=True, nullable=False)
+    reviewer_languages = db.relationship('ReviewLanguage', secondary=reviewer_languages,
+                                         lazy='subquery', backref=db.backref('pages', lazy=True))
```

Comment:
I think `'pages'` here comes from the example doc, but this should probably be `'reviewers'` to define a backref of `ReviewLanguage.reviewers`

Reply:
yep